src/comparison/goal.rs

```diff
@@ -11,7 +11,8 @@ use ordered_float::OrderedFloat;
 /// The default name of the goal comparison.
 pub const NAME: &str = "Goal";
 
-const WEIGHT: f64 = 0.9375;
+const WEIGHT: f64 = 0.75;
```

Comment:
I changed the weight to match the other algorithms, such as median and average. I think we simply forgot to change this back when we lowered the value for the others. We might want to change this in the original LiveSplit too.